src/logic/hooks/useEstimateTransactionGas.tsx

```diff
@@ -157,7 +157,6 @@ export const useEstimateTransactionGas = ({
             txRecipient,
             txAmount: txAmount || '0',
             operation: operation || CALL,
-            safeTxGas,
```

Comment:
Is this on purpose? I haven't understood this change

Reply:
Yes, we were sending that parameter but we don't really need it any more. In that function we should get just the backend estimation.
https://github.com/gnosis/safe-react/pull/2193/files#diff-5711367a44d6d99049597d88399c2b489e222124a118fc2909f4e2986cb3577dL172

In case we can't get it we are already handling it in `useEstimateTransactionGas` setting it to 0 or assigning the manual value.

This was legacy code when we set a manual safeTxGas, so in that case we ensured that manual value was assigned at any cost. Now the advanced parameters work differently